.gitattributes

```diff
@@ -0,0 +1 @@
+*.csr eol=lf
```

Comment:
Not a huge deal, but in general we should consistently add newlines to the ends of our file. Some scripts/parsers don't like not having it